modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -10145,6 +10145,9 @@ object-action-trigger[onAfterUpdate]=On After Update
 object-definition-data=Object Data
 object-definition-id=Object ID
 object-definition-table-name=Table Name
+object-entry-value-exceeds-integer-field-size=Object entry value exceeds Integer field allowed size.
```

Comment:
Could you fix the language key with all the words contained in the message?

Reply:
Sure!
Thanks for the tip!